### PR TITLE
feat: grants/rsk

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

This PR integrates R-BTC and RSK ERC20 tokens to the grant product
User can create RSK grant and accept donations in all of RSK tokens 

- [x] Grant Creation
- [x] Update Grant Details
- [x] Contribute Grant
  - [x] R-BTC
  - [x] ERC-20
- [x] Backend logic sync
  - [x] R-BTC
  - [x] ERC-20 
  
image

##### Testing

- [x] create grant + verify + add to cart - https://www.loom.com/share/164051ee4c9f404e8ad86f25fbcd4beb  
- [x] checkout R-BTC + RDOC - https://www.loom.com/share/dbce3bc3c230419e9735000c2f8b9182
- [x] sync pending contributions - https://www.loom.com/share/5e5a63ccd89746538629c1a0fdec8fc9

### CODE REVIEW

File: app/assets/v2/js/grants/cart/rsk_extension.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+          'tenant': 'RSK',
+          'comment': grant.grant_comments,
+          'amount_per_period': grant.grant_donation_amount
+        }]
+      };
+
+      const apiUrlBounty = `v1/api/contribute`;
+
+      fetchData(apiUrlBounty, 'POST', JSON.stringify(payload)).then(response => {
+        console.log(response);
```
Comment (chibie) [inline app/assets/v2/js/grants/cart/rsk_extension.js:116]: you can remove the logs here

File: app/assets/v2/js/grants/funding.js [hunk 2]
```diff
@@ -191,6 +191,9 @@ function tokenOptionsForGrant(grant) {
   } else if (grant.tenants && grant.tenants.includes('KUSAMA')) {
     tokenDataList = tokenDataList.filter(token => token.chainId === 59);
     tokenDefault = 'KSM';
+  } else if (grant.tenants && grant.tenants.includes('RSK')) {
+    tokenDataList = tokenDataList.filter(token => token.chainId === 30);
+    tokenDefault = 'RSK';
```
Comment (chibie) [inline app/assets/v2/js/grants/funding.js:196]: tokenDefault should be R-BTC

**Review by chibie — CHANGES_REQUESTED** [review]

File: app/assets/v2/js/grants/funding.js (hunk 2 above)
Comment (thelostone-mc) [reply app/assets/v2/js/grants/funding.js:196]: Oh fuck ! nice catch man

File: app/assets/v2/js/grants/cart/rsk_extension.js (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/js/grants/cart/rsk_extension.js]: done

**Review by chibie — APPROVED** [review]

**Pull request merged**